http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```diff
@@ -794,9 +796,10 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
     for {
       nodepool <- defaultNodepool
       defaultClusterName <- KubernetesNameUtils.getUniqueName(KubernetesClusterName.apply)
+      clusterName = aksClusterName.map(c => KubernetesClusterName(c.value)).getOrElse(defaultClusterName)
```

Comment:
This will populate the _actual_ cluster name in the DB for AKS clusters.

Out of scope is migrating existing records in the DB. I don't think a liquibase migration will work for this because it will require querying the cluster's LZ. I was imagining a standalone script to pull data out of the LZ and migrate existing records.

The only impact to not migrating existing records is they won't have these new metrics in Grafana..